admin_guide/high_availability.adoc

```diff
@@ -242,6 +243,101 @@ The list of command options described here are a subset that are relevant to thi
 Each VIP in the set may end up being served by a different node.
 ====
 
+[[check-notify]]
+=== Check and Notify scripts
+
+The *keepalived* port monitoring feature uses a simple port check ( </dev/tcp/<ip>/<watch-port> ) to verify that the application is running. The admin can provide a script that does whatever additional verification is needed, for example, the script can test a web server by issuing a request and verifying the response. The script must exit with 0 for PASS and 1 for FAIL.
```

Comment:
how about "uses a simple tcp connect to verify"

And should we say that if watch-port is 0 the test is skipped?